plutus-ledger-api/plutus-ledger-api.cabal

```diff
@@ -47,6 +47,7 @@ common lang
     -fno-strictness
 
 common ghc-version-support
+  -- This is only used for the plugin tests.
```

Comment:
I was wondering why `plutus-ledger-api` depends on the plugin, but it's only for some tests, so I've added a reminder about that.

Reply:
Yes exactly.